src/test/java/net/openhft/chronicle/queue/impl/single/StoreAppenderTest.java

```diff
@@ -58,15 +59,15 @@ public void threadDump() {
     @Test
     public void clearUsedByThreadThrowsUnsupportedOperationException() throws IOException {
         try (SingleChronicleQueue queue = SingleChronicleQueueBuilder.single(queueDirectory.newFolder()).build()) {
-            assertThrows(UnsupportedOperationException.class, () -> queue.acquireAppender().singleThreadedCheckReset());
+            assertThrows(UnsupportedOperationException.class, () -> acquireThreadLocalAppender(queue).singleThreadedCheckReset());
         }
     }
 
     @Ignore("net/openhft/chronicle/queue/channel/PublishHandler.java:25")
     @Test
     public void resetUsedByThreadThrowsUnsupportedOperationException() throws IOException {
         try (SingleChronicleQueue queue = SingleChronicleQueueBuilder.single(queueDirectory.newFolder()).build()) {
-            assertThrows(UnsupportedOperationException.class, () -> queue.acquireAppender().singleThreadedCheckReset());
+            assertThrows(UnsupportedOperationException.class, () -> acquireThreadLocalAppender(queue).singleThreadedCheckReset());
```

Comment:
These tests are kind of redundant now, I don't even know why they're still in there. Please delete. They were put there to try and prevent people passing thread-local appenders between threads, but someone disabled them later on.